exercises/practice/complex-numbers/.meta/src/reference/java/ComplexNumber.java

```diff
@@ -44,14 +46,6 @@ ComplexNumber conjugate() {
     }
 
     ComplexNumber exponentialOf() {
-        return new ComplexNumber(Math.cos(imaginary), Math.sin(imaginary)).multiply(Math.exp(real));
-    }
-
-    private ComplexNumber divide(double factor) {
-        return new ComplexNumber(real / factor, imaginary / factor);
-    }
-
-    private ComplexNumber multiply(double factor) {
-        return new ComplexNumber(factor * real, factor * imaginary);
+        return new ComplexNumber(Math.exp(real) * Math.cos(imaginary), Math.exp(real) * Math.sin(imaginary));
     }
-}
+}
```

Comment:
Could you make check that there is an empty line at the end? Without it, [Checkstyle fails](https://github.com/exercism/java/actions/runs/11643029850/job/32423316068?pr=2852#step:4:680).

Reply:
Sure, I'll do it right now!